pkgs/applications/networking/maestral/default.nix

```diff
@@ -1,24 +1,38 @@
-{ stdenv, lib, python3Packages, fetchFromGitHub
-, withGui ? false, wrapQtAppsHook ? null }:
+{ stdenv
+, lib
+, python3Packages
```

Comment:
in general, you should import the interpreter itself, and access the packages through python.pkgs